en/app.ftl

```diff
@@ -36,6 +36,12 @@ nav-profile-contact = Contact us
 # This is only visible to Premium users.
 nav-profile-contact-tooltip = Get in touch about { -brand-name-relay-premium }
 nav-profile-image-alt = { -brand-name-firefox-account(capitalization: "uppercase") } Avatar
+nav-dashboard = Dashboard
+nav-phone = Phone Number
+nav-duo-description = Switch dashboards
+nav-duo-email-mask-alt = Email masks
+nav-duo-phone-mask-alt = Phone masks
+
```

Comment:
These strings are for the switcher between the email and phone dashboards on mobile:

![image](https://user-images.githubusercontent.com/4251/196904319-8440ffd6-d1db-48e9-abd2-105c954e8cd8.png)